doc/cygnus-ngsi/installation_and_administration_guide/configuration_examples.md

```diff
@@ -1,47 +1,470 @@
-#<a name="top"></a>Cygnus configuration examples
+# <a name="top"></a>Cygnus configuration examples
 Content:
 
-* [Single source, single storage (basic configuration)](#section1)
-* [Single source, multiple storages](#section2)
-* [Single source, single storage, parallel sinking](#section3)
-* [Single source, multiple storages, parallel sinking](#section4)
-* [Multiple sources](#section5)
-* [Using interceptors](#section6)
+* [Basic configurations](#section1)
+    * [Simplest scenario](#section1.1)
+    * [Scenarios involving multiple persistence backends](#section1.2)
+* [Advanced configurations](#section2)
+    * [Scenarios involving multiple FIWARE services](#section2.1)
+        * [Multiple agents](#section2.1.1)
+        * [Single agent, multiple sources](#section2.1.2)
+        * [Single agent, single source, multiplexing per FIWARE service](#section2.1.3)
+        * [Orion Context Broker subscriptions](#section2.1.4)
+    * [Scenarios involving multiple FIWARE service paths](#section2.2)
+    * [Scenarios involving a pool of sinks per persistene backend](#section2.3)
 
-##<a name="section1"></a>Single source, single storage (basic configuration)
-To be done
+## <a name="section1"></a>Basic configurations
+Basic configurations are those involving a single listener/source and one or more sinks receiving a copy of the same internal NGSI event obtained once a NGSI notification is processed at the source.
+
+### <a name="section1.1"></a>Simplest scenario
+A Cygnus agent based on a single source, a single channel and a single sink.
+
+```
++-----------------------------------+
+|  +-----+     +----+     +------+  |
+|  | src |-----| ch |-----| sink |  |
+|  +-----+     +----+     +------+  |
++-----------------------------------+
+```
+
+Its agent configuration file is the usal one:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1420/commits/ea511e5c58c65fc69304692013f33e82ae986f1a